packages/cursorless-engine/src/processTargets/modifiers/EveryScopeStage.ts

```diff
@@ -73,20 +73,13 @@ export class EveryScopeStage implements ModifierStage {
     if (scopes == null) {
       // If target had no explicit range, or was contained by a single target
       // instance, expand to iteration scope before overlapping
-      try {
-        scopes = this.getDefaultIterationRange(
-          scopeHandler,
-          this.scopeHandlerFactory,
-          target,
-        ).flatMap((iterationRange) =>
-          getScopesOverlappingRange(scopeHandler, editor, iterationRange),
-        );
-      } catch (error) {
-        if (!(error instanceof NoContainingScopeError)) {
-          throw error;
-        }
-        scopes = [];
-      }
+      scopes = this.getDefaultIterationRange(
+        scopeHandler,
+        this.scopeHandlerFactory,
+        target,
+      ).flatMap((iterationRange) =>
+        getScopesOverlappingRange(scopeHandler, editor, iterationRange),
+      );
```

Comment:
This basically reverts the logic added in https://github.com/cursorless-dev/cursorless/pull/2081/files#diff-a47f14c27291690e54ca5b2e034c62193e9da0615bdb7b3be4153ede7668914f.\

It was necessary when there was this fallback code after it:

```
    if (scopes.length === 0) {
      if (scopeType.type === "collectionItem") {
        // For `collectionItem`, fall back to generic implementation
        return this.modifierStageFactory
          .getLegacyScopeStage(this.modifier)
          .run(target);
      }

      throw new NoContainingScopeError(scopeType.type);
    }
```

We have since removed the legacy code but didn't know that this code also needed to be removed